components/event-publisher-proxy/pkg/cloudevents/builder/generic_test.go

```diff
@@ -76,6 +84,13 @@ func Test_Build(t *testing.T) {
 			givenApplicationName: "appName1",
 			wantError:            true,
 		},
+		{
+			name:                 "should return error if empty source",
+			givenSource:          "",
+			givenType:            "order.created.v1",
+			givenApplicationName: "appName1",
+			wantError:            true,
+		},
```

Comment:
Is this only for v1alpha2 CRD? Also what if in case of `exact`? As in some cases source does not matter. ([reference](https://github.com/kyma-project/kyma/issues/15618#issuecomment-1310364282))

Reply:
it is only for v1alpha2 jetstream. There is no `exact` in EPP right?

Reply:
As discussed, the `exact` flow in EPP does not go through this function.